packages/gatsby-recipes/recipes/storybook-ts.mdx

```diff
@@ -0,0 +1,68 @@
+# Add Storybook
+
+[Storybook](https://storybook.js.org/) is an open source tool for developing UI components in isolation for React, Vue, and Angular. It makes building stunning UIs organized and efficient.
+
+This recipe adds support for JavaScript and TypeScript components.
+
+---
+
+Install TypesScript + babel plugins and presets as well as the Storybook React NPM packages and addons
+
+<NPMPackage name="typescript" />
+<NPMPackage name="babel-loader" />
+<NPMPackage name="@babel/preset-react" />
+<NPMPackage name="@babel/preset-env" />
+<NPMPackage name="@babel/preset-typescript" />
+<NPMPackage name="@babel/plugin-proposal-class-properties" />
+<NPMPackage name="babel-plugin-remove-graphql-queries" />
+<NPMPackage name="react-docgen-typescript-loader" />
+
+<NPMPackage name="@storybook/react" />
```

Comment:
Should this change also be made to JavaScript [Recipe](https://raw.githubusercontent.com/gatsbyjs/gatsby/master/packages/gatsby-recipes/recipes/storybook-js.mdx) too?

Reply:
Ugh sorry I forgot there was already another storybook recipe — does it make sense to add two? One for both TypeScript & JavaScript? Perhaps just combine the two?

But yeah either way, we should combine the install steps.

Reply:
I was wondering about that myself. The _potential_ problem with combining the two is that `typescript` needs to be added as a dependancy. JavaScript only projects might not want TypeScript added because it's not being used, same story with the `@babel/preset-typescript` and `react-docgen-typescript-loader` seems unnecessary to add these deps to non TypeScript projects. 

I'm happy to make the change to the `js` recipe so the two are aligned if you like?

Reply:
yeah — those are heavy enough dependencies that keeping things seperate makes sense — cool, yeah let's leave them seperate but apply the changes to the JS recipe as well

Reply:
Sweet. I'll push the changes to the `js` recipe to this PR if that's cool?